settings.gradle

```diff
@@ -48,6 +48,10 @@ include "beam-runners-gearpump"
 project(":beam-runners-gearpump").dir = file("runners/gearpump")
 include "beam-runners-google-cloud-dataflow-java"
 project(":beam-runners-google-cloud-dataflow-java").dir = file("runners/google-cloud-dataflow-java")
+include "beam-runners-google-cloud-dataflow-java-examples"
+project(":beam-runners-google-cloud-dataflow-java-examples").dir = file("runners/google-cloud-dataflow-java/examples")
```

Comment:
FWIW this long names are just a maven hack as I understand it. Mapping them here saves us ~2 lines in the publishing config where we'd have to set the artifactId. So for things that aren't published, the best name for them is no name and gradle automatically treats a directory `foo/baz/bizzle` as the project `foo:baz:bizzle`.